clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1894,6 +1994,25 @@ public int getLastTime(String event) {
         return -1;
     }
 
+    /**
+     * Retrieves last occurrence timestamp of event associated with the current user/{@link CleverTapAPI#getCleverTapID(OnInitCleverTapIDListener) CleverTap ID}.
+     * This operation involves a database query and should be called from a background thread.
+     * <br>
+     * Example usage:
+     * <br>
+     * <code>
+     * // Call from background thread <br>
+     * long lastTs = getUserEventLogLastTs("navigation_clicked")
+     * </code>
+     *
+     * @param eventName Name of the event to get last timestamp for (e.g., "navigation_clicked", "item_selected")
+     * @return Last occurrence timestamp of the event for current user, or -1 if the event does not exist or there was an error
```

Comment:
Why do we return -1 if there are no records? Shouldn't we return 0 if there are no records and return -1 only if there is an error?

Reply:
resolved by @vasct